scripts/python/update-user-defined-attribute-type.py

```
@@ -0,0 +1,79 @@
+attributes_to_update = []
+mapping_settings_to_update = []
+
+for workspace in workspaces:
+    sage_intacct_credentials = SageIntacctCredential.objects.filter(workspace_id=workspace.id).first()
+    if sage_intacct_credentials:
+        try:
+            sage_intacct_connection = SageIntacctConnector(credentials_object=sage_intacct_credentials, workspace_id=workspace.id)
+            dimensions = sage_intacct_connection.connection.dimensions.get_all()    
+            for dimension in dimensions:
+                if dimension['objectName'] not in default_destination_attributes and dimension['userDefinedDimension'] == 'true':
+                    if dimension['objectName'] != dimension['objectLabel']:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)